pyproject.toml

```diff
@@ -13,3 +13,10 @@ version = "0.1.0"
 
 [tool.poetry.dependencies]
 python = "3.11.*"
+
+
+[tool.poetry.group.dev.dependencies]
```

Comment:
This group should be optional, because we don't want these dependencies to be installed by default with:
```python
poetry install
```

We want to install this group only where we actually need it, like this
```python
poetry install --with dev
```